plugins/node/opentelemetry-instrumentation-express/src/express.ts

```diff
@@ -54,6 +54,14 @@ export class ExpressInstrumentation extends InstrumentationBase<
     );
   }
 
+  getConfig(): ExpressInstrumentationConfig {
```

Comment:
I have seen you add this mechanism to other instrumentations already. If this is a feature we want on all instrumentations we should implement it in the base class so it doesn't need duplicated implementations all over the place. If you think this is a feature all instrumentations should have please create an issue in core so it can be discussed.

Reply:
https://github.com/open-telemetry/opentelemetry-js/issues/2174

Reply:
As it's not published yet, I'm keeping `getConfig` until then